### PR TITLE
Feature/in 910 virus scan

### PR DESCRIPTION
**Pull request by lorenzoAtBEIS**
Replacing ClamAV virus scanner with Cloudmersive virus scanner api. 
**This PR contains a free developer API key for dev purposes only**

### CODE REVIEW

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — APPROVED** [review]

File: INZFS.MVC/Services/FileUpload/FileUploadService.cs
Comment (sunilbeis) [inline INZFS.MVC/Services/FileUpload/FileUploadService.cs:88]: Can't this return a boolean value instead of text ?

Comment (sunilbeis) [reply INZFS.MVC/Services/FileUpload/FileUploadService.cs:88]: Or is that if a string is returned, then the file has some issue, in which case the string can be simply returned as it.

File: INZFS.MVC/Services/VirusScan/VirusScanService.cs
Comment (sunilbeis) [inline INZFS.MVC/Services/VirusScan/VirusScanService.cs:31]: I think, this list could change depending uopn the file. Does it matter ?

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — DISMISSED** [review]

**Review by sunilbeis — APPROVED** [review]

**Review by sunilbeis — APPROVED** [review]

**Comment by lorenzoAtBEIS** [comment]
Waiting for DSU approval - closing for now

**Pull request closed without merging**
